### ISSUE TITLE
Test performance

### ISSUE THREAD

**Issue by tcw888** [pouya-bashivan/EEGLearn#5]
The code "eeg_cnn_lib.py" looks like: it loads data from "Sample data" folder, converts the data to 2D images, then create 5 models, but doesn't train the data.  Do you have code for training and test set prediction?

**Comment by pbashivan** [comment]
I need to clean up the training script and add some comments. I should be able to do so in few days.

**Comment by pbashivan** [comment]
9f930351df02c3b72137b9ae70d0a661716a1442
